Updated support email ids, added additional images, and some small improvements. (#2094)
DataDog/integrations-extras: seagence/README.md

## Overview

[Seagence][1] is a production-grade Realtime Defect Detection and Resolution tool for Java applications. Using ExecutionPath technology, Seagence detects known and unknown defects caused by various issues like multithreading issues, swallowed, handled, and unhandled exceptions, and others, including defects that are disguised in a 200 success HTTP response code.

With this integration, Seagence backend continuously analyzes the data stream from the Seagence agent to detect defects when they occur, including the root cause of the defect. When a defect is detected, the integration will send an event to Datadog to alert your team. Using the out-of-the-box dashboard, you have visibility into the detected defects and root causes to eliminate debugging and troubleshooting. More details about the defect can be found on [SeagenceWeb][2].

## Setup

### Installation

Visit [Seagence][1] to sign up for free. Once registered, navigate to the Seagence tile on the [Datadog Integrations page][5] and click **Install Integration**. Click **Connect Accounts** on the tile, which guides you through the Datadog OAuth2 flow to grant Seagence access to post Events to your Datadog account.

> After connecting accounts, go to "Assets" tab on the tile > Recommended Monitors > Click on "Seagence Defect Detection Monitor". This will redirect you to create the out-of-the-box monitor. Click "Create" at the bottom of the page to install the Seagence monitor.

Reviewer: ```suggestion
After connecting your accounts, go to the "Assets" tab on the tile. Click **Recommended Monitors** > **Seagence Defect Detection Monitor**. This will redirect you to create the out-of-the-box monitor. Click **Create** at the bottom of the page to install the Seagence monitor.
```

To remove the Datadog integration from Seagence:
1. Uninstall the integration from Datadog by clicking **Uninstall Integration**. Once you uninstall the integration, any previous authorizations are revoked.
2. Ensure that all API keys associated with the integration have been disabled by searching for the integration name on the [API Keys Management page][6].

> 3. Delete the associated monitor by going into Monitors > Manage Monitors > Hover mouse on "Seagence Defect Detection Monitor" > Click "Delete" button.

Reviewer: ```suggestion
3. Delete the associated monitor by going to **Monitors** > **Manage Monitors**. Hover over **Seagence Defect Detection Monitor** and click **Delete**.
```
